web/src/search/input/MonacoQueryInput.tsx

```diff
@@ -69,7 +69,40 @@ function addSouregraphSearchCodeIntelligence(
     return subscriptions
 }
 
-const NOOP_KEYBINDINGS = [Monaco.KeyMod.CtrlCmd | Monaco.KeyCode.KEY_F, Monaco.KeyMod.CtrlCmd | Monaco.KeyCode.Enter]
+/**
+ * HACK: this interface and the below type guard are used to free default Monaco
+ * keybindings (such as cmd + F, cmd + L) by unregistering them from the private
+ * `_standaloneKeybindingService`.
+ *
+ * This is necessary as simply registering a noop command with editor.addCommand(keybinding, noop)
+ * prevents the default Monaco behaviour, but doesn't free the keybinding, and thus still blocks the
+ * default browser action (eg. select location with cmd + L).
+ *
+ * See upstream issues:
+ * - https://github.com/microsoft/monaco-editor/issues/287
+ * - https://github.com/microsoft/monaco-editor/issues/102 (main tracking issue)
+ */
+interface MonacoEditorWithKeybindingsService extends Monaco.editor.IStandaloneCodeEditor {
+    _actions: {
+        [id: string]: {
+            id: string
+            alias: string
+            label: string
+        }
+    }
+    _standaloneKeybindingService: {
+        addDynamicKeybinding(keybinding: string): void
+    }
+}
+
+const hasKeybindingService = (
+    editor: Monaco.editor.IStandaloneCodeEditor
+): editor is MonacoEditorWithKeybindingsService =>
+    hasProperty('_actions')(editor) &&
+    typeof editor._actions === 'object' &&
```

Comment:
Lodash has this function to exclude ES6 Maps
```suggestion
    isPlainObject(editor._actions) &&
```

Reply:
Done in d1bce59